draco/asp/examples/histogram.lp

```diff
@@ -10,6 +10,7 @@ entity(view,root,(v,0)).
 entity(mark,(v,0),(m,0)).
 entity(encoding,(m,0),(e,0)).
 attribute((encoding,field),(e,0),(f,0)).
+attribute((encoding,binning),(e,0),10).
```

Comment:
Why do we need to specify the binning here>? Shouldn't it be automatically be added by generate?